physionet-django/console/static/console/css/console.css

```diff
@@ -1,5 +1,9 @@
 /* Based on sb-admin.css from start-bootstrap */
 
+.application-ok {
```

Comment:
I think using "traffic" in the style name might be clearer. Instead of filling a cell with color, you could render a circle (this makes the lights easier to reuse elsewhere too).

```css
.traffic-circle {
  display: inline-block;
  border: rgb(99, 99, 99) solid 1px;
  width: 19px;
  height: 19px;
  border-radius: 10px; /* half of width + borders */ 
}

.traffic-red {
  background: rgb(204, 50, 50);
}

.traffic-orange {
  background: rgb(231,180,22);
}

.traffic-green {
  background: rgb(45, 201, 55);
}
```

Reply:
Added the centering to the circle to lie in the middle of the cell container.